src/Functions.Worker.Extensions.OpenAI/Startup.cs

```diff
@@ -3,9 +3,11 @@
 
 using System.Text.Json;
 using Microsoft.Azure.Functions.Worker.Core;
-using Microsoft.Azure.Functions.Worker.Extensions.OpenAI;
 using Microsoft.Azure.Functions.Worker.Extensions.OpenAI.Embeddings;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Azure.Functions.Worker.Extensions.OpenAI;
```

Comment:
using statements should be sorted.

In Visual Studio, you can right click - `Remove and sort usings` or `Ctrl+R, Ctrl+G`

Reply:
done!